javascript/chronometer.js

```diff
@@ -1,34 +1,53 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
-  }
+  
+    this intervalID = setInterval(() => {
+      this.currentTime = intervalCounter += 1;
+      
+      if(callback){
+      return callback;
+      }
+
+    }, 1000);
+   
+   
+    }
+}
 
   getMinutes() {
-    // ... your code goes here
+  
+    let Minutes = Math.floor this.currentTime /60;
+    return getMinutes;
+
   }
 
   getSeconds() {
-    // ... your code goes here
-  }
+   
+    let numberofSeconds = this.currentTime % 60;
+    return getSeconds;
+}
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if(value < 10 ) return 0
+    return value;
   }
 
   stop() {
-    // ... your code goes here
+    clearInterval this.intervalId;
```

Comment:
clearInterval, like Math.floor, is a function. In order to run it you have to use parenthesises.